File: user_input/JavaScript/message.js
// use js message.js to run
// Written in node
// Takes string input from user and writes to console.
const readline = require("readline");
const rl = readline.createInterface({ input: process.stdin, output: process.stdout });

rl.question("Enter some text:\n", function(myText) {
    console.log(`Here is your text:\n${myText}`);
    rl.close();
});

rl.on("close", function() {
    process.exit(0);
});

File: user_input/Scala/message.scala
// Compile with scalac message.scala and run with scala message (omit .class)
// Takes string input from user and writes to console.

import scala.io.StdIn.readLine

object message extends App {
  println("Enter some text: ")
  val myText = readLine()

  println(s"Here is your text back:\n$myText")
}
